bin.src/makeGeom.py

```diff
@@ -236,7 +234,7 @@ def printElectronics(buff, ccdId, ccdname, xidx, yidx, infoA, infoB):
 
     # 0th layer is pure metadata
     for ccd in range(1, len(ptr)):
-        #for ccd in [1, 28]:
+        # for ccd in [1, 28]:
```

Comment:
Consider deleting commented-out code. This looks even safer to remove than the bit earlier.